src/Extensions/BlockArchiveExtension.php

```diff
@@ -48,7 +48,7 @@ public function getArchiveField()
             'Breadcrumbs' => function ($val, $item) {
                 $parent = $item->Page;
 
-                return $parent ? $parent->Breadcrumbs() : null;
+                return (($parent && method_exists($parent, 'Breadcrumbs')) ? $parent->Breadcrumbs() : null);
```

Comment:
```suggestion
                return ($parent && $parent->hasMethod('Breadcrumbs')) ? $parent->Breadcrumbs() : null;
```

This would allow extensions to provide the Breadcrumbs method as well as the base class